lib/importers/article_importer.rb

```diff
@@ -60,4 +66,44 @@ def import_articles_from_title_query(results)
     end
     Article.import articles, on_duplicate_key_update: [:title, :namespace]
   end
+
+  ###############
+  # API methods #
```

Comment:
This is all about interacting with the MediaWiki API, independently of ArticleImporter, so it probably should not be part of this class. It should be either in WikiApi or in a new WikiPostApi.